2_segmenting_data/segmentation_utils.py

```diff
@@ -0,0 +1,251 @@
+# from cellpose.io import logger_setup
+from cellpose import models, core, io, utils
+from typing import Tuple
+
+import pathlib
+import pandas as pd
+import skimage.io as io
+
+import cv2
+import numpy as np
+
+import matplotlib.path as mplPath
+
+
+def get_outlines(image: np.ndarray, model_specs: dict) -> pd.DataFrame:
+    """finds outlines of objects using cellpose specs from model_specs and return pandas array with outlines of objects
+    Args:
+        image (np.ndarray): image with objects to segment
+        model_specs (dict): specifications for cellpose segmentation
+    Returns:
+        pd.DataFrame: dataframe with object outlines
+    """
+    # Empty list for the outlines data to be appended to
+    objects_data = []
+
+    # CellPose model specs based on the specs for the object
+    cellpose_model = models.Cellpose(gpu=True, model_type=model_specs["model_type"])
```

Comment:
kinda nitpicky comment, but `gpu` should be a default function argument set to True. (that is, instead of hardcoding `gpu=True` inside the function.

Reply:
Fixed!